src/modules/RootPart.rb

```diff
@@ -1711,7 +1712,20 @@ def MountPartitions(root_device_current)
 
               Builtins.y2milestone("fstab %1", fstab)
 
-              if !(
+              if reiserfs_in_fstab?(fstab)
+                reiserfs_entries = fstab.find_all {|e| e["vfstype"].include? "reiser" }
```

Comment:
np: I would go for `select` instead of `find_all`. There are no convention on our style guide, but it is [common practice in the Ruby community](https://github.com/bbatsov/ruby-style-guide).